app/views/content/fund-your-training/_questions.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="category__questions__container">
+
+  <section class="container category__questions">
+
+    <div class="category__question">
+      <h2>Want your own personal adviser?</h2>
+
+      <p>
+        Teacher training advisers provide free, one-to-one support by phone, text or email.
+        They can support you to find funding for your training.
+      </p>
+
+      <a href="/teacher-training-advisers" class="button">Get an adviser</a>
+    </div>
+
+    <div class="category__question">
+      <h2>Do you have a question about funding?</h2>
```

Comment:
```suggestion
      <h2>Have a question about funding?</h2>
```
These two sections don't seem to line up properly - would suggest dropping the 'do you' to keep it all on one line

Reply:
Or "Got a question...."?

Agree it should be shorter - happy with either